components/automate-ui/src/app/modules/infra-proxy/reset-admin-key/reset-admin-key.component.ts

```diff
@@ -0,0 +1,75 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { Store } from '@ngrx/store';
+import { FormGroup, Validators, FormBuilder } from '@angular/forms';
+import { NgrxStateAtom } from 'app/ngrx.reducers';
+import { Regex } from 'app/helpers/auth/regex';
+import { Subject, combineLatest } from 'rxjs';
+import { LayoutFacadeService, Sidebar } from 'app/entities/layout/layout.facade';
+import { filter, takeUntil } from 'rxjs/operators';
+import { EntityStatus, pending } from 'app/entities/entities';
+import { AdminKey } from 'app/entities/reset-admin-key/reset-admin-key.model';
+import {
+   updateStatus
+} from 'app/entities/reset-admin-key/reset-admin-key.selectors';
+import { UpdateAdminKey } from 'app/entities/reset-admin-key/reset-admin-key.actions';
+
+@Component({
+  selector: 'app-reset-admin-key',
+  templateUrl: './reset-admin-key.component.html',
+  styleUrls: ['./reset-admin-key.component.scss']
+})
+
+export class ResetAdminKeyComponent implements OnInit {
```

Comment:
You have defined `isDestroyed` but are never using it; the class must implement OnDestroy.